recipes/delete-me/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "giotto-learn" %}
+{% set version = "0.1a" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/matteocao/{{ name }}.git
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  script:
+    - git submodule update --init --recursive
+    - "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - gcc  # [not win]
+    - vs2017  # [win]
```

Comment:
If you need ``c++`` compilers it should be
```
    - {{ compiler('cxx') }}
```
and you can remove ``gcc`` and ``vs2017``
please see:
https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=compiler%20cxx#compilers